client/styles/components/_nav.scss

```diff
@@ -39,16 +39,27 @@
   height: 100%;
 
   & button {
-    padding: #{12 / $base-font-size}rem #{10 / $base-font-size}rem;
+    padding: 0 #{5 / $base-font-size}rem;
```

Comment:
i don't think these changes to the Nav are really helping that much. i'd prefer to have a drill down situation here. and maybe add another toolbar or something that's stuff that folks would want to do a lot on mobile, i.e. examples, open.

Reply:
The current changes were to prevent overlap of nav-items and logo below 576 till 390, if a new nav is required for below 576, I can revert the changes

Idea for the nav:
 - The logo, login button/user account can remain in the first nav, 
 - A second nav containing file, edit, sketch, help and examples nav items (or their respective icons) can be implemented while the toolbar can be put beneath them. 
 - The toolbar can have current sketch information, name and alongside can be a dropdown containing options for full preview, full code editor view or combined view.

A second implementation may include using a hamburger menu, but the content to be used in it needs to be discussed as it might hinder the discoverability and usage for users but it may also be a good alternative for responsive navbar.
@catarak could you suggest something?

Reply:
hmmm. i think i need to set aside some time to really think about what is necessary for mobile going forward—the design just isn't there yet and it's hard to say yes or no to this. i thought it made sense to think about just rendering sketches on mobile at first, but it seems like there's a big demand for writing code on mobile as well.